home/models.py

```diff
@@ -1,3 +1,84 @@
 from django.db import models
 
 # Create your models here.
+class Privilege(models.Model):
+    view_transaction = models.BooleanField()
+    create_transaction = models.BooleanField()
+    authorize_transaction = models.BooleanField()
+    transaction_type = models.CharField(max_length=20)
+    issue_check = models.BooleanField()
+    fund_transfer = models.BooleanField()
+    view_account = models.BooleanField()
+    create_account = models.BooleanField()
+    modify_account = models.BooleanField()
+    close_account = models.BooleanField()
+    delete_account = models.BooleanField()
+    account_type = models.CharField(max_length=20)
+    request_type = models.CharField(max_length=20)
+    access_logs = models.BooleanField()
+    user_type = models.CharField(max_length=10)
+
+    def __str__(self):
+        return self.user_id
+        
+class User(models.Model):
+    first_name = models.CharField(max_length=20)
+    last_name = models.CharField(max_length=20)
+    business_name = models.CharField(max_length=50)
+    street_address = models.CharField(max_length=20)
+    city = models.CharField(max_length=20)
+    state = models.CharField(max_length=20)
+    zip_code = models.IntegerField()
+    mobile_number = models.CharField(max_length=10)
+    email = models.CharField(max_length=20)
+    birthdate = models.DateTimeField()
+    ssn = models.CharField(max_length=9)
+    user_type = models.CharField(max_length=10)
+    joining_date = models.DateTimeField()
+    password = models.CharField(max_length=200)
+    privilege_id = models.ForeignKey(Privilege, on_delete=models.CASCADE)
+
+    def __str__(self):
+        if self.first_name:
+            return self.first_name
+        else:
+            return self.business_name
+
+class Account(models.Model):
```

Comment:
How are credit cards handled? Don't we need stuff like credit card number and so on? Probably we should check with TA and commit this for now?

Reply:
I do not believe we need to have actual cards. What I see in the document is talking about debiting or crediting an account. Meaning, just moving money in or out.